src/Surfnet/Stepup/Configuration/Value/AllowedSecondFactorMap.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Copyright 2017 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\Stepup\Configuration\Value;
+
+use Surfnet\StepupBundle\Value\SecondFactorType;
+
+final class AllowedSecondFactorMap
```

Comment:
I'm not sold on this concept. More importantly though: no matter the usage or implementation,, it is most certainly not part of the domain, it must be moved out of the domain.